src/chmod/chmod.rs

```diff
@@ -40,7 +42,17 @@ pub fn uumain(args: Vec<String>) -> i32 {
     opts.optflag("R", "recursive", "change files and directories recursively");
     opts.optflag("h", "help", "display this help and exit");
     opts.optflag("V", "version", "output version information and exit");
-    // TODO: sanitize input for - at beginning (e.g. chmod -x testfile).  Solution is to add a to -x, making a-x
+    // sanitize input for - at beginning (e.g. chmod -x testfile). Remove
+    // the option and save it for later, after parsing is finished.
+    let re = Regex::new("^-{1}([rwx]|[0-7])").unwrap();
```

Comment:
By the way, there are more dependencies that can be removed from chmod (eg. memchr, aho-corasick). I'll update the pull request when we agree on the regex part.